tests/test_add_meta_tag.py

```diff
@@ -4,17 +4,18 @@
 import snare
 import shutil
 import configparser
+from utils.page_path_generator import generate_unique_path
 
 
 class TestAddMetaTag(unittest.TestCase):
 
     def setUp(self):
-        if not os.path.exists("/opt/snare/pages/test"):
-            os.makedirs("/opt/snare/pages/test")
+        self.main_page_path = generate_unique_path()
+        os.makedirs(self.main_page_path)
         self.content = '<html><head>title</head><body>sample</body></html>'
-        self.page_dir = "test"
+        self.page_dir = self.main_page_path[-9:]
```

Comment:
you can use `rsplit` here. we should avoid "magic constants" or make it clear from context. here it is not obvious why it's 9